src/translations/en.json

```
@@ -3836,6 +3836,14 @@
               "advanced_options": "Advanced options",
               "external_activation": "Allow external activation of remote control",
               "external_activation_secondary": "Allows you to turn on remote control from your Nabu Casa account page, even if you're outside your local network",
+              "strict_connection": "Restrict access to logged in users",
+              "strict_connection_secondary": "When a user is not logged in to your Home Assistant instance, they will not be able to access your instance remotely",
+              "strict_connection_mode": "Mode",
+              "strict_connection_modes": {
+                "disabled": "Disabled",
+                "static_page": "Error page",
```

```suggestion
                "static_page": "Static page",
```

I would call it static page to be consistent with the non-cloud wording

---

Or Unauthorized page for both? For Static page you have to known a bit about development.

---

I'm not sure if we should use `Unauthorized` as strict connection is before our authorization layer.

---

Couple of alternatives:
- Block page
- Guard page
- Safeguard page

---

As we concluded on Discord: Guard page.